### PR TITLE
Fb coin flip

### PR DESCRIPTION
**Pull request by mkendall07**
This PR moves ~50% of the traffic to non-secure endpoint per FB request.

### CODE REVIEW

File: adapters/facebook.go [hunk 1]
```diff
@@ -49,8 +51,21 @@ type facebookParams struct {
 	PlacementId string `json:"placementId"`
 }
 
+func coinFlip() bool {
+	random := rand.Intn(2)
+	if random == 0 {
+		return false
+	}
+	return true
+}
```
Comment (nhedley) [inline adapters/facebook.go]: Any reason we can't shorten this to `return rand.Intn(2) != 0`?

**Comment by nhedley** [comment]
Just want to confirm since `rand.Seed(time.Now().UnixNano())` is called in `pbs_light.go` there's no need to call it again?

**Comment by abramovic** [comment]
right, we only need to call ``rand.Seed`` once

File: adapters/facebook.go (hunk 1 above)
Comment (mkendall07) [reply adapters/facebook.go]: nope, good suggestion.
